packages/backend/services/ticket/task.ts

```diff
@@ -181,7 +211,13 @@ const taskServiceTicket = new TaskService(
                 const thirdPartyToken = connection.tp_access_token;
                 const tenantId = connection.t_id;
 
-                if (thirdPartyId !== TP_ID.bitbucket && !fields && fields && !fields.listId) {
+                if (
+                    thirdPartyId !== TP_ID.bitbucket &&
+                    thirdPartyId !== TP_ID.github &&
+                    !fields &&
+                    fields &&
+                    !fields.listId
```

Comment:
Ain't this `!fields && fields` conflict each other in **&&** . 

```suggestion
                    !fields &&
                    fields &&
                    !fields.listId
```
```suggestion
                if (
                    thirdPartyId !== TP_ID.bitbucket &&
                    thirdPartyId !== TP_ID.github && (
                    !fields ||
                    fields &&
                    !fields.listId
                    )
```